paradise.dme

```diff
@@ -5,6 +5,220 @@
 // END_INTERNALS
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "config"
```

Comment:
It seems with your last commit, you accidentally added a bunch of stuff to the dme, and also made a change in the example config toml file.

Reply:
Oh whoops, that's a testing change. Lemme fix that really quick. I have to revert them every time I push. I did it earlier with the "complicated" commit.